server/backbone/api/parent.py

```diff
@@ -344,14 +353,19 @@ def modify_quest(qid, title, description, reward, due, timestamps):
 
         {
           "data": {
-            "completed_on": "",
-            "description": "You're going on a quest to save the princess, brush your teeth.",
-            "due": 1559131200.0,
-            "id": 10,
-            "next_occurrence": 1559131200.0,
-            "recurring": true,
-            "reward": 2,
-            "title": "Brush your teeth"
+            "completed_now": false,
+            "lvled_up": false,
+            "qst": {
+              "completed_on": "Sat, 06 Jul 2019 21:11:35 GMT",
+              "description": "You're going on a quest to save the princess, brush your teeth so you don't embarrass yourself.",
+              "due": 1559250000.0,
+              "id": 1,
+              "needs_verification": true,
+              "recurring": false,
+              "reward": 12,
+              "title": "This is the initial quest",
+              "verified_on": null
```

Comment:
is `verified_on` useful for anything? can we just turn `needs_verification` to false

Reply:
in child.py I just see `verified`, I don't think we need `verified_on`

Reply:
`verified_on` is how you tell that something was accepted by a parent. needs verification says whether the quest needed verification at all.